static/skin/i18n.js

```diff
@@ -1,10 +1,22 @@
 import mustache from '../skin/mustache.min.js?KIWIXCACHEID'
 
 const Translations = {
+  defaultLanguage: null,
   default: null,
   current: null,
 
   load: function(lang, asDefault=false) {
+    if ( asDefault ) {
+      this.defaultLanguage = lang;
+      this.loadTranslationsJSON(lang, asDefault);
+    } else if ( lang == this.defaultLanguage ) {
+      this.current = 'use default';
```

Comment:
No, but it doesn't matter since this code is gone in later refactoring (as I mentioned in https://github.com/kiwix/libkiwix/pull/846#issuecomment-1328199508:

> history of changes related to translation loading needs to be cleaned up

)